opcua/client/client.py

```diff
@@ -125,8 +125,8 @@ def __init__(self, url, timeout=4):
         self.product_uri = "urn:freeopcua.github.io:client"
         self.security_policy = ua.SecurityPolicy()
         self.secure_channel_id = None
-        self.secure_channel_timeout = 3600000  # 1 hour
-        self.session_timeout = 3600000  # 1 hour
+        self.secure_channel_timeout = session_timeout
+        self.session_timeout = secure_channel_timeout
```

Comment:
there is no reason to make thes properties available as argument. Just set the properties